### PR TITLE
Add start up info modal

### LINKED ISSUE #605: Start up Password info Pop up
An info Icon should be added at the top of the `Startup password` toggle button on the settings page.
When the info button is clicked is should show so basic message detailing what the start up password is about.
image

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #605 
This pr also adds warning modal to user agent and specific peers when the toggle is turned off.

image

image

### CODE REVIEW

File: ui/page/settings_page.go [hunk 1]
```diff
@@ -391,8 +415,15 @@ func (pg *SettingsPage) Handle() {
 		pg.wal.SaveConfigValueForKey(dcrlibwallet.BeepNewBlocksConfigKey, pg.beepNewBlocks.IsChecked())
 	}
 
-	for pg.changeStartupPass.Clicked() {
+	if pg.infoButton.Button.Clicked() {
+		info := modal.NewInfoModal(pg.Load).
+			Title("Set up startup password").
+			Body("Startup password help protect your wallet from unauthorized access.").
```
Comment (dreacot) [inline ui/page/settings_page.go]:
```suggestion
			Body("Startup password helps protect your wallet from unauthorized access.").
```

File: ui/page/settings_page.go [hunk 2]
```diff
@@ -479,7 +510,10 @@ func (pg *SettingsPage) Handle() {
 			pg.showSPVPeerDialog()
 			return
 		}
-		pg.WL.MultiWallet.DeleteUserConfigValueForKey(specificPeerKey)
+
+		title := "Remove specific peer"
+		msg := "Are you want to proceed with removing the specific peer?"
```
Comment (dreacot) [inline ui/page/settings_page.go]:
```suggestion
		msg := "Are you sure you want to proceed with removing the specific peer?"
```

File: ui/page/settings_page.go [hunk 3]
```diff
@@ -498,7 +532,10 @@ func (pg *SettingsPage) Handle() {
 			pg.showUserAgentDialog()
 			return
 		}
-		pg.WL.MultiWallet.DeleteUserConfigValueForKey(userAgentKey)
+
+		title := "Remove user agent"
+		msg := "Are you want to proceed with removing the user agent?"
```
Comment (dreacot) [inline ui/page/settings_page.go]:
```suggestion
		msg := "Are you sure you want to proceed with removing the user agent?"
```

**Review by dreacot — CHANGES_REQUESTED** [review]

**Review by dreacot — APPROVED** [review]

File: ui/page/settings_page.go [hunk 4]
```diff
@@ -479,7 +510,10 @@ func (pg *SettingsPage) Handle() {
 			pg.showSPVPeerDialog()
 			return
 		}
-		pg.WL.MultiWallet.DeleteUserConfigValueForKey(specificPeerKey)
+
+		title := "Remove specific peer"
+		msg := "Are you sure you want to proceed with removing the specific peer?"
+		pg.showWarningModalDialog(title, msg, specificPeerKey)
```
Comment (beansgum) [inline ui/page/settings_page.go:516]: Is there any need for adding a prompt here? It doesn't look necessary to confirm from the user before removing persistent peer.

**Review by beansgum — CHANGES_REQUESTED** [review]

File: ui/page/settings_page.go (hunk 4 above)
Comment (Sirmorrison) [reply ui/page/settings_page.go:516]: I am looking at the point where a user accidentally clicks on the button. The peer is removed almost immediately and the user would need to set it up the second time.

**Review by beansgum — APPROVED** [review]

**Pull request merged**